src/models/device.ts

```diff
@@ -350,6 +360,10 @@ const getDeviceModel = function (
 			);
 		}
 
+		if (options == null) {
+			options = {};
+		}
```

Comment:
Use `options: { runDetached?: boolean } = {}` above to provide a default argument and avoid the need for this
```suggestion
```